src/Disk.php

```diff
@@ -327,8 +344,9 @@ public function getPublishResources($limit = 20, $offset = 0)
 			}
 
 			return [];
-		}))
-			->setLimit($limit, $offset);
+		};
+
+		return (new {$this->collectionClass}($callback))->setLimit($limit, $offset);
```

Comment:
PHP Parse error:  syntax error, unexpected token "{" in \src\Disk.php on line 349